site/content/en/docs/tasks/monitor_pending_workloads/pending_workloads_on_demand.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Pending Workloads on-demand"
+date: 2023-12-04
+weight: 3
+description: >
+#   Monitor pending workloads.
+---
+
+This page shows you how to monitor pending workloads with VisibilityOnDemand feature.
+
+The intended audience for this page are [batch administrators](/docs/tasks#batch-administrator).
+
+From version v0.6.0, Kueue provides the ability for a batch administrators to monitor
+the pipeline of pending jobs, and help users to estimate when their jobs will
+start.
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool has communication with your cluster.
+- [Kueue is installed](/docs/installation) in version v0.6.0 or later.
+
+## Enabling feature VisibilityOnDemand
+
+VisibilityOnDemand is an `Alpha` feature disabled by default, check the [Change the feature gates configuration](/docs/installation/#change-the-feature-gates-configuration) section of the [Installation](/docs/installation/) for details.
+
+## Monitor pending workloads on demand
+
+> _Available in Kueue v0.6.0 and later_
+
+To install a simple setup of cluster queue, run the following command:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kueue/main/site/static/examples/admin/single-clusterqueue-setup.yaml
+```
+
+To install two local queues, run the following command:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kueue/main/site/static/examples/admin/localqueues-setup.yaml
+```
```

Comment:
I believe that we should provide the only minimum case. So setting a localqueue by `single-clusterqueue-setup.yaml` should be enough.

Or is there any reason that we should have multiple localqueues?

Reply:
Yeah,  I don't have a strong view, maybe we can go with minimal setup in the first iteration.

One reason to have 2 localqueues is to demonstrate that the positions in ClusterQueue might be interleaving between two LocalQueues, for this purpose we could just install another local queue from a dedicated file.

Reply:
My intention was to demonstrate two facts:
1) The position of Workloads across LocalQueues is consistent, 
2) A user requesting Workloads belonging to one LocalQueue does not have any information about pending Workloads in other LocalQueues.

Reply:
Sounds reasonable to me, WDYT @tenzen-y ?